src/test/scala/io/iohk/ethereum/jsonrpc/JsonRpcControllerSpec.scala

```diff
@@ -1323,6 +1323,39 @@ class JsonRpcControllerSpec extends FlatSpec with Matchers with PropertyChecks w
     ))
   }
 
+  it should "eth_getAccountRecentTransactions" in new TestSetup {
```

Comment:
Minor detail: Maybe we could also add a test `EthServiceSpec` testing how `eth_getAccountRecentTransactions` works without mocking it?

Reply:
done